### PR TITLE
FileSystem: added move()

### PR DESCRIPTION
**Pull request by hermajan**
- bug fix? no
- new feature? yes
- BC break? no
- doc PR: nette/docs#680

I added FileSystem::move() method for moving files.

### CODE REVIEW

File: src/Utils/FileSystem.php [hunk 1]
```diff
@@ -88,6 +88,17 @@ public static function delete(string $path): void
 	}
 
 
+	/**
+	 * Moves a file or directory.
+	 * @throws Nette\IOException
+	 */
+	public static function move(string $source, string $dest, bool $overwrite = true): void
+	{
+		static::copy($source, $dest, $overwrite);
```
Comment (dg) [inline src/Utils/FileSystem.php]: Vhodnější je použít rename(), protože copy() je mnohem pomalejší.

Comment (nechutny) [reply src/Utils/FileSystem.php]: Better is try rename and then if it fail (eg. two filesystems) try copy.

**Comment by dg** [comment]
What is a difference between FileSystem::move() and FileSystem::rename()?

**Comment by dg** [comment]
ping @hermajan

**Comment by hermajan** [comment]
I did this function because from the source code and documentation I didn't know that FileSystem::rename() function moves files. Because it looks like the function of this pull request is unnecessary, I would suggest it might be worth mentioning explicitly in the documentation that FileSystem::rename() can be used to move files or make a FileSystem::move() alias of rename() for moving of files.

**Comment by dg** [comment]
Ok, so I close it.

**Pull request closed without merging**